### ISSUE TITLE
错误是因为学习率太大，导致跨过了最小值

### ISSUE THREAD

**Issue by SkySailing** [hungduyle/CS231n-2018Spring#2]

**Comment by dhungle** [comment]
Hi,

I am not sure I understand your issue. I used Google translate to interprete the title, but apparently I am still missing something. Would you mind giving me a context (which exercise) and a description (in English preferably)?

**Comment by SkySailing** [comment]
sorry.there is no issue.i notice your comment that the net don't work in the softmax .so i recommend that maybe the learning rate is too high. because i see the similar problem.
